gcp_variant_transforms/libs/bigquery_schema_descriptor.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A dict based description for BigQuery's schema."""
+
+from __future__ import absolute_import
+
+from collections import namedtuple
+from apache_beam.io.gcp.internal.clients import bigquery  # pylint: disable=unused-import
+
+__all__ = ['SchemaDescriptor']
```

Comment:
I guess this is a convention adopted for VT (all classes have it). Otherwise, it's not relevant here (at least for now). I have added it here for consistency.

Reply:
I prefer if we drop this if we don't really need it. For example, I did not add it in [processed_variant](https://github.com/googlegenomics/gcp-variant-transforms/blob/master/gcp_variant_transforms/libs/processed_variant.py) module because I did not see any reason for it, although I realized it is in other modules. Adding @arostamianfar to make sure there is no specific reason we are missing.

Reply:
Talked offline with Asha. No particular reason to keep it, removed it.